CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+ - Additional exception handling in _connect of consumer
```

Comment:
Strictly speaking should be "Add additional exception handling in `_connect` of consumer"